board/knot/raspberrypi/device_table.txt

```diff
@@ -3,4 +3,5 @@
 # <name>				<type>	<mode>	<uid>	<gid>	<major>	<minor>	<start>	<inc>	<count>
 /usr/local/bin/knot-fog-source		r	755	1003	1003	-	-	-	-	-
 /usr/local/bin/knot-fog-connector	r	755	1003	1003	-	-	-	-	-
+/usr/local/rabbitmq-server	        r	755	1004	1004	-	-	-	-	-
```

Comment:
About commit message:
Besides, put the PR number I'd prefer to put commit hash

Reply:
Done.